### ISSUE TITLE
BUG?: Class 'MicrosoftAzure\Storage\Common\ServicesBuilder' not found 

### ISSUE THREAD

**Issue by in-session** [macopedia/typo3-azurestorage#13]
Hallo, ich wollte deine EXT mal testen, nach dem einrichten des Ordners erhalten ich aber einen Fehler.
Ich habe mal ein paar Screenshorts gemacht mit der Fehlerbeschreibung, sowie die Daten die eingegeben wurden. Grüße

Typo3 8.7.9
PHP Version 7.1.9

![ashampoo_snap_2018 01 27_11h15m00s_014_](https://user-images.githubusercontent.com/16542619/35471043-65f91964-0354-11e8-9356-7b876c115360.png)
![ashampoo_snap_2018 01 27_11h09m04s_012_](https://user-images.githubusercontent.com/16542619/35471045-68cfae0a-0354-11e8-8e6c-8380f11f12d4.png)
![ashampoo_snap_2018 01 27_11h08m32s_011_chrome legacy window](https://user-images.githubusercontent.com/16542619/35471046-6b6db6c0-0354-11e8-8472-6f95ccc44833.png)

**Comment by jooler** [comment]
I meet this also,
but when I use "composer install ..."
it works
in the typo3 backend ->extensions -> get extensions -> install AzureStorage
I see "'MicrosoftAzure\Storage\Common\ServicesBuilder' not found "

but it's composer mode and lose some modules,
so please , Can you fix it?

**Comment by benjaminhirsch** [comment]
Hello and sorry for the late reply @in-session . I will investigate this. :-)

**Comment by jooler** [comment]
very thanks for your work!

**Comment by benjaminhirsch** [comment]
I've released a new version of the extension today. This new version uses the latest version of the Azure Storage Blob libray and related to this, some minor code changes. Unfortunatley I'm still unable to reproduce the mentioned behavior. 

Could you please do me a favor and try the new version? Who knows, maybe the problem is fixed due to the new library 😄

**Comment by jooler** [comment]
thank you very much,I will try it now.

**Comment by jooler** [comment]
its not work , please see this
![2018-10-23 2 42 12](https://user-images.githubusercontent.com/25960405/47311832-5aec1200-d66d-11e8-944f-3068384a437d.png)

**Comment by jooler** [comment]
but the old version is ok if I make something,
1. install extension;

2.copy azure storage sdk to src/vendor, like this
![2018-10-23 2 44 45](https://user-images.githubusercontent.com/25960405/47312621-6cceb480-d66f-11e8-8981-677188b091e3.png)


3.edit this file /src/vendor/composer/autoload_psr4.php add 2 line
`'MicrosoftAzure\\Storage\\' => array($vendorDir . '/microsoft/azure-storage/src'),
    'B3N\\AzureStorage\\TYPO3\\' => array($baseDir . '/typo3conf/ext/azurestorage/Classes'),`

![2018-10-23 2 48 09](https://user-images.githubusercontent.com/25960405/47312631-77894980-d66f-11e8-9863-963a143e6a46.png)


4.edit file /src/vendor/composer/autoload_static.php add this

![2018-10-23 2 50 15](https://user-images.githubusercontent.com/25960405/47312648-807a1b00-d66f-11e8-90ca-3e151d101ee3.png)


until now , it works.

**Comment by benjaminhirsch** [comment]
To be honest, I'm not quite sure what's going on here. 😕 Actually you don't have to bother with those kind of things, because composer does all the magic. All you have to do is `composer require b3n/azurestorage` and that's it. If you have already installed the old version, simply change the version number to `^0.5.1` in your composer.json and the run `composer update` All neccessary librarys will automatically be installed.

I've tested it here again with a completley new installation, everything is working as expected. Hmmm

**Comment by jooler** [comment]
OK let me try again.

**Comment by jooler** [comment]
I'm very sorry that my English is not good enough.
I mean, after installing TYPO3, download the distribution version.

Download the installation files from get.typo3.org and install it .
Download https://github.com/benjaminhirsch/typo3-azurestorage/archive/0.5.2.zip and  into the Typo3 backend, install it , and these error messages will appear

But with the composer command, everything is perfect.

**Comment by benjaminhirsch** [comment]
Ah, now I understand. Unfortunatley this package isn't intended to work without composer. If you need for any reason to install it manually, you have to take care of installing the dependencies and autoloading by yourself.

**Comment by jooler** [comment]
Okay, thank you for your work. This extension is really useful to me. I'm very happy to find a way to get it to work. Thank you again.
